docs/development.rst

```diff
@@ -17,7 +17,7 @@ Contributions
 
 Some guidance for contributors:
 
-- discuss about changes on github issue tracker, IRC or mailing list
+- discuss changes on the GitHub issue tracker, IRC or on the mailing list
```

Comment:
"mailing list" - "on the" is repeated for that, but not for "IRC".

so, guess we do not need to repeat it for "mailing list" either.